### PR TITLE
Create kobo_guide_V1.md

### PR DESCRIPTION
**Pull request by S-r-f-l**
Kobo Guide version 1

### CODE REVIEW

File: docs/kobo_guide_V1.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Using KoboToolbox for Geographical Data Collection
+
+## Guide Introduction:
+
+Welcome to our comprehensive guide on using KoboToolbox for gathering geographical data. In today's digital age, efficient data collection is crucial for various projects, from research initiatives to humanitarian efforts. KoboToolbox offers a powerful platform for designing, deploying, and managing surveys and forms, with a special emphasis on geo data collection.
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:5]: Sounds pretty much like taken from their website ? Could you add a link ? I like the [Cheat Sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

File: docs/kobo_guide_V1.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+This Introduction aims to harness the capabilities of KoboToolbox to gather geo data for variaty of projects. By following the steps outlined in this guide, you will be equipped to use the provided .xls file and deploy it for data collection in the field. , [(and analyze the collected geographical data to derive meaningful insights.)]
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:7]: Double parenthesis is too much at the end.
Which xls file is meant ?
Could you also state that the file is an [XLSForm ](https://xlsform.org/en/)

File: docs/kobo_guide_V1.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+So, if you're ready to embark on a journey of efficient geo data collection using KoboToolbox, let's dive in and explore the steps involved in harnessing the full potential of this versatile tool.
+
+**Reminder:* Please Download the followind Excel file before continuing with the steps to use the KoboToolbox:
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:11]: Best to work with the link syntax instead of bare link IMHO...

File: docs/kobo_guide_V1.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ 
+5. **Deploy Forms:**
+   - Once your form is ready, click on the "Deploy" button.
+   - Choose the deployment method (e.g., web form, mobile app) and follow the instructions to deploy your form.
+
+6. **Collect Data:**
+   - If using the web form, share the form link with respondents. If using the mobile app, download the KoboCollect app on Android devices and log in with your account.
+   - Collect data from respondents using the deployed form either online or offline.
+
+7. **Regular Maintenance:**
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:47]: I think that we can skip this part for the moment...

File: docs/kobo_guide_V1.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - Regularly check your account for updates, review data quality, and make necessary adjustments to your forms and projects based on feedback and evolving requirements.
+
+8. **Backup and Security:**
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:50]: We can also skip this for the moment I would say...

File: docs/kobo_guide_V1.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    - Ensure data security by regularly backing up your data and following best practices for data management and protection.
+
+Remember, this guide provides a basic overview of using KoboToolbox. For more advanced features and functionalities, refer to the official documentation and resources provided by KoboToolbox.
```
Comment (fretchen) [inline docs/kobo_guide_V1.md:53]: We can also remove this.

**Review by fretchen — CHANGES_REQUESTED** [review]
I think that this is a nice starting point. I already added a number of comments where fitting. 

- [ ] could you update the mkdocs.yml to add this to the documentation ?
- [ ] Could you  add a "note" at the beginning which mentions that this is still an early phase and we are testing out the workflow ?
- [ ] rename the file to kobo_guide.md ?

File: docs/kobo_guide.md [hunk 7]
```diff
@@ -1,17 +1,18 @@
+**Disclaimer:* This guide is in the early phase of development, and we are currently testing out the workflow. Please bear with us as we refine the content and improve the workflow. Your feedback and suggestions are greatly appreciated as we work towards enhancing this guide.
```
Comment (fretchen) [inline docs/kobo_guide.md:1]: I would remove the disclaimer here.

**Review by fretchen — CHANGES_REQUESTED** [review]
Almost there...

File: docs/kobo_guide.md (hunk 7 above)
Comment (S-r-f-l) [reply docs/kobo_guide.md]: done

**Pull request merged**